style.css

```diff
@@ -0,0 +1,976 @@
+  background-color: rgba(0, 0, 0, 0.85);
+  z-index: 1;
+  color: #fff;
+  width: 100%;
+  height: auto;
+}
+
+.activity-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.activity-container h2 {
+  width: 100%;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  font-weight: 500;
+  font-size: 25px;
+  margin: 20px 0;
+  padding: 10px 0;
+}
+
+.active {
+  margin: 10px 20px;
+  padding: 10px 20px;
+  display: flex;
+  flex-direction: row;
+  background: #63666d;
+}
+
+.active img {
+  width: 70px;
+  height: auto;
+  align-items: center;
+  margin-top: 2%;
+}
+
+.active p {
+  align-items: flex-start;
+  font-family: 'Lato', sans-serif;
+  font-size: 15px;
+  font-weight: 400;
+}
+
+.show {
+  display: none;
+}
+
+.love {
+  width: 70%;
+  justify-content: center;
+  height: 80px;
+  color: #fff;
+  background: #ec5242;
+  text-align: center;
+  margin: 40px 70px;
+  padding: 20px 10px;
+  font-family: 'Lato', sans-serif;
+  font-size: 24px;
+  cursor: pointer;
+}
+
+.space {
+  width: 46px;
+  height: 2px;
+  align-self: center;
+  background-color: rgba(236, 82, 66, 1);
+  margin-top: -25px;
+  margin-bottom: 30px;
+}
+
+.space3 {
+  width: 46px;
+  height: 2px;
+  align-self: center;
+  background-color: rgba(236, 82, 66, 1);
+  margin-top: -25px;
+  margin-bottom: 30px;
+  margin-left: 45%;
+}
+
+.maka-d {
+  padding-left: 8px;
+  display: flex;
+  flex-direction: column;
+}
+
+.maka-d:nth-child(even) {
+  display: none;
+}
+
+.maka-box {
+  padding-left: 8px;
+  display: flex;
+  flex-direction: row;
+}
+
+.onye-okwu {
+  display: flex;
+  flex-direction: row;
+  margin: 0 10px;
+  padding: 10px 10px;
+}
+
+.onye-okwu img {
+  width: 150px;
+  height: 150px;
+  padding: 0 0;
+}
+
+.okwu {
+  margin-left: 5%;
+  margin-right: -10px;
+  padding: -30px 12px;
+  width: 100%;
+}
+
+.okwu h2 {
+  margin-bottom: 7px;
+  font-family: "Lato", sans-serif;
+  font-size: 22px;
+  font-weight: 700;
+}
+
+.okwu p {
+  width: 100%;
+  opacity: 1;
+}
+
+.head-us {
+  margin-bottom: 10px;
+}
+
+.first-speak {
+  width: 100%;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  font-weight: 500;
+  font-size: 25px;
+  margin: 20px 0;
+  padding: 10px 0;
+}
+
+.btn {
+  width: 90%;
+  justify-content: center;
+  height: 50px;
+  color: #000;
+  background: #fff;
+  text-align: center;
+  margin: 40px 25px;
+  padding: 10px 10px;
+  font-family: 'Lato', sans-serif;
+  font-size: 16px;
+  cursor: pointer;
+  border: 1px solid #d3d3d3;
+}
+
+.btn:hover {
+  background-color: #63666d;
+  color: #fff;
+  border-radius: 0;
+}
+
+.ukwu-cont {
+  background-color: #272a31;
+}
+
+.ukwu-head h2 {
+  width: 100%;
+  text-align: center;
+  font-family: 'Lato', sans-serif;
+  font-weight: 500;
+  font-size: 30px;
+  margin: 20px 0;
+  padding: 40px 0 10px 0;
+  color: #fff;
+}
+
+.ukwu-box {
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  gap: 20px;
+  margin: 40px 0;
+  padding: 30px 0;
```

Comment:
- [ ] Kindly make sure you stick to the design as much as possible. please consider adding this overlay on these images on the `about` page as is in the design.
`Hint: You may add the linear gradient to the background if the image was added background image`
Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/217934773-c4b7e544-4f7a-4e08-9fee-1e41c9c25c9d.png)

Figma template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/217935180-0bd573cb-fe05-4a94-83fe-b1745f828ca6.png)